VSTestHost/VSTestHost.csproj

```diff
@@ -138,6 +160,18 @@
     <Content Include="Resources\Package.ico" />
     <None Include="source.extension.vsixmanifest" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Any reason for the bootstrapper packages?  Or did they just get added automatically by VS during upgrade?